### DISCUSSION TITLE
which github-tools do you use for your project mgmt?

### DISCUSSION THREAD

**Discussion by fsj-digital** [community/community#53144] [New to GitHub]
good day dear community

which github-tools do you use for your project mgmt?

Organizations have been using GitHub Issues for project management for some time. GitHub Issues is essentially a system for describing issues,for tracking their progress and communicating around them.

Milestones: These can be used, for example, to track the progress of groups of issues or simply track them via pull requests in a project. When you create a milestone, effectively sets a "due date" for any issues/pull requests you associate with it.

Wikis: Every GitHub repository has a section for hosting dateb and also documentation called a wiki.

Automated triggers: A success factor of the GitHub project management system is that it also contains so-called triggers that automatically move tasks.

Issues: this is mainly used for bug tracking, but its core functionality means that it is also a useful tool for more general project management purposes.

which github-tools do you use for your project mgmt. Are there some tools out there that work like trello for example?

i want to get to know more github-tools.

**Comment by NJacksonDev** [comment]
In additional to the ones you mentioned, there are a couple others which may be helpful, including one with Trello (listed below).

**GitHub Projects**: a Kanban-style project management tool that allows you to organize your issues and pull requests into customizable boards. You can create columns for different stages of your workflow, and move issues between columns as they progress.

**GitHub Power-Up for Trello**: Trello offers a GitHub Power-Up that allows you to connect your GitHub repositories to Trello boards. With this Power-Up, you can see your GitHub issues and pull requests directly in Trello, and you can link Trello cards to GitHub issues.
